pkgs/by-name/ae/aegisub/0005-drop-scintilla-ime.patch

```diff
@@ -0,0 +1,327 @@
+diff --git a/src/meson.build b/src/meson.build
+index 1c97c0c85..b01ad5e90 100644
+--- a/src/meson.build
```

Comment:
Could you try sending this patch upstream?

Reply:
It's difficult, upstream once [did so](https://github.com/arch1t3cht/Aegisub/commit/c487dd2bdbab58ad9f50da6624132165abd7a123) for it use private symbols link and so not compatible with wxWidgets dist, but then the repo's owner [revert it](https://github.com/arch1t3cht/Aegisub/commit/6dda04da9fd9b43a22266fdb1c4fa293b418e3db) because this breaks IME input in darwin.
I will abandon this patch and try to compile wxWidgets locally in Darwin to fix it.